docs/apis/cli.md

```diff
@@ -187,6 +187,8 @@ Action "run" compiles and runs a program.
                                           java.net.URLClassLoader}.
      -d,--detached                        If present, runs the job in detached
                                           mode
+     -D,--configDir </path/to/confdir>    The configuration directory with which
```

Comment:
`-D`? I think that can be easily confused. We don't have to have a short option.